test/parallel/test-url-format-invalid-input.js

```diff
@@ -3,17 +3,20 @@ require('../common');
 const assert = require('assert');
 const url = require('url');
 
-// https://github.com/nodejs/node/pull/1036
-const throws = [
-  undefined,
-  null,
-  true,
-  false,
-  0,
-  function() {}
-];
-for (let i = 0; i < throws.length; i++) {
-  assert.throws(function() { url.format(throws[i]); }, TypeError);
+const throwsObjsAndReportTypes = new Map([
+  [undefined, 'undefined'],
+  [null, 'null'],
+  [true, 'boolean'],
+  [false, 'boolean'],
+  [0, 'number'],
+  [function() {}, 'function'],
+  [Symbol('foo'), 'symbol']
+]);
+
+for (const [obj, type] of throwsObjsAndReportTypes.entries()) {
```

Comment:
Nit: `throwsObjsAndReportTypes` can be used directly instead of calling `entries()`.